packages/react-sdk/src/tree/session-storage-polyfill.tsx

```diff
@@ -0,0 +1,35 @@
+// This is a temporary work around for https://github.com/remix-run/remix/issues/3659
```

Comment:
uhh, this looks fishy: consumers will think sessionStorage works, but it basically just shifts the problem to warnings instead of error. Shouldn't consumers be testing if sessionStorage is available before using it?

Reply:
what is currently using sessionStorage?

Reply:
Remix uses it to restore scroll position. They need to fix it on their side. Hopefully they'll do it as the issue is already open, then we can remove this workaround.